include/ofi_mr.h

```diff
@@ -242,6 +245,7 @@ struct ofi_mr_cache_params {
 	size_t				max_size;
 	char *				monitor;
 	int				cuda_monitor_enabled;
+	int				rocr_monitor_enabled;
```

Comment:
We need a better way to handle this.  This cleanup can come in a follow set of changes though.